MIP0/mip0.md

```diff
@@ -7,287 +7,249 @@ Title: The Maker Improvement Proposal Framework
+  - A MIP has been superseded or deprecated.
+  - A MIP has been deferred for over six months.
+  - A MIP Author has abandoned the proposal and no person has communicated willingness to take over the responsibility of a MIP Author.
 
-**Deferred:** when a proposal has been deemed as not ready or not a priority but can be re-proposed at a later date.
--   Request for Comments (RFC) - Forum poll/signal request rejects a MIP Proposal.
+#### MIP Status Change Process
 
-**Obsolete:** when a proposal is no longer used or is out of date, such as:
-    
--   A MIP is replaced with a new proposal.
--   A MIP has been deferred for over 6 months.
--   A MIP Author has abandoned the proposal and no person has communicated willingness to take over MIP Author responsibility.
--   A MIP has been replaced by a newer, more updated MIP Proposal.
--   A MIP no longer makes sense to keep in consideration.
-    
-  
-**MIP Status Change Process:**
-    
+If a MIP Author requests a status change for a MIP, a MIP Editor will review it. If the status change is warranted, the MIP Editor will change the status. Otherwise, the MIP Editor will revert and highlight issues for the MIP Author to fix before requesting another status change.
 
-A status change for a MIP is requested by the MIP Author and will be reviewed by the MIP Editor(s) to see if it meets the status criteria of the requested status change. If it does, the Editor(s) will change the status of the MIP and the Author may proceed with the next stage of the process. If it does not, the MIP Editor(s) will revert with highlighted issues, and the Author must fix the highlighted issues before requesting another status change.
-    
 ---
 ### MIP0c4: MIP Components and MIP Component Types
 
 
-**MIP Components**
+#### MIP Components
+
+In order to satisfy the Core Principles of clarity and completeness, a MIP may need to have multiple components that each define distinct units of logic.
+
+> MIP Components follow the naming convention `MIPXcY`, where `X` is the parent MIP and `Y` is the component number.
+
+##### Component Types
+
+A MIP Component can be of one of two types: General or Process. The type of General MIP Components is usually left undeclared: It is the assumed type of components that are not of Process type.
+
+#### Process MIP Component  
+
+MIP Components of the Process type shape a specific process flow for the Maker community to adopt and standardize with respect to how governance operates. This MIP component type helps streamline particular processes in a transparent, open, and traceable manner.
+
+In this MIP, `MIP0c6`, `MIP0c12` and `MIP0c13` are Process Components.
+
+Process MIP Components require no special template.
+
+#### Subproposals
```

Comment:
Probably want:
1. What are subproposals.
2. Process for subproposals.
3. What process MIPs define for subproposals.
4. Naming convention for subproposals.

Reply:
This is much better now. Good job.